app/javascript/article-form/components/__tests__/ArticleCoverImage.test.jsx

```diff
@@ -247,21 +242,19 @@ describe('<ArticleCoverImage />', () => {
 
       it('displays an uploading message', async () => {
         const onMainImageUrlChange = jest.fn();
+        /* eslint-disable no-unused-vars  */
         const { container, findByText } = render(
           <ArticleCoverImage
             mainImage=""
             onMainImageUrlChange={onMainImageUrlChange}
           />,
         );
-        const nativeInput = container.querySelector(
-          '#native-cover-image-upload-message',
-        );
+        /* eslint-enable no-unused-vars  */
 
         // Fire a change event in the hidden input with JSON payload for an error
-        const fakeUploadingMessage = JSON.stringify({ action: 'uploading' });
-        fireEvent.change(nativeInput, {
-          target: { value: fakeUploadingMessage },
-        });
+        const fakeUploadingMessage = { action: 'uploading', 'namespace': 'coverUpload' };
+        const event = createEvent('ForemMobile', document, { detail: fakeUploadingMessage }, { EventType: 'CustomEvent' });
+        fireEvent(document, event);
```

Comment:
These tests are giving me a pretty hard time. I'm trying to fire the CustomEvents, which should be handled by the EventListener inside the component, but I can't seem to make these work properly.

@nickytonline do you have any suggestions to work around this?

I was also forced to add `eslint-disable no-unused-vars` because I was no longer using `container` but I don't know the correct syntax this should use because things break when I just remove the `container` variable from here (the props returned by the `render`). If I keep the `container` then `eslint` complains that there's an unused defined variable.

This last bit is mostly me not being familiar with Preact/Jest 😅